apps/expo/src/components/EventTab.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Text, View } from "react-native";
+
+import type { RouterOutputs } from "~/utils/api";
+
+type GetActivityOutput = RouterOutputs["activity"]["getSchedule"][number];
+
+interface EventProps {
+  activity: GetActivityOutput;
+  attending?: boolean;
+}
+
+export default function EventTab({ activity, attending }: EventProps) {
+  return (
+    <View className={`w-fill bg-p-95 flex-col justify-center rounded-md p-4`}>
+      <Text className=" text-p-10 font-poppins500 text-base">
```

Comment:
for all of your text classes, use the predefined classes from #212

Reply:
Ok will do!